app/views/estimate.scala.html

```diff
@@ -0,0 +1,84 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.main_template
+@import utils.ImplicitCurrencyFormatter._
+@import utils.ImplicitDateFormatter._
+@import views.html.helpers.accordionHelper
+@import models.calculation.EstimatesViewModel
+@import models.core.breadcrumb.{Breadcrumb, BreadcrumbItem}
+@import views.html.helpers.breadcrumbHelper
+@import views.helpers.EstimatesPage
+@import views.html.helpers.calcBreakdownHelper
+@import views.html.helpers.backToHomeHelper
+
+@(model: EstimatesViewModel, taxYear: Int)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig, user: auth.MtdItUser[_])
```

Comment:
Would you put `taxYear` inside of the view model rather than as a separate parameter in the view? Seems to make more sense to me.